wee_slack.py

```diff
@@ -4011,8 +4027,13 @@ def command_reply(data, current_buffer, args):
     channel = EVENTROUTER.weechat_controller.buffers[current_buffer]
     try:
         msg_id, text = args.split(None, 1)
+        if msg_id == "-broadcast":
+            msg_id, text = text.split(None, 1)
+            broadcast = True
+        else:
+            broadcast = False
     except ValueError:
-        w.prnt('', 'Usage: /reply <count/id> <message>')
+        w.prnt('', 'Usage: /reply [-broadcast] <count/id> <message>')
```

Comment:
You should add `[-broadcast]` to the docstring some lines above as well (I wasn't allowed to make a comment at that line).